package.json

```diff
@@ -15,6 +15,7 @@
     "define:change": "changeset",
     "release:apply": "changeset version",
     "release:publish": "changeset publish",
+    "lint": "npx @biomejs/biome check --write --unsafe ./src",
```

Comment:
It is not necessary to use `npx @biomejs/biome` just `biome`, the script should be `“lint”:“biome check --write --unsafe ./src”,`

Reply:
Fixed!